### PR TITLE
Refactor to TMDB multi-search

### LINKED ISSUE #478: 🧼 Refactor search results to use multi search TMDB query
We can make a single call on SearchResults instead of one for movie and one for tv

https://developer.themoviedb.org/reference/search-multi

### PR DESCRIPTION
**Pull request by Thenlie**
Updated the search results screen to use a single query to get both movie and tv show results. I also took the opportunity to update the helper export structure by adding an index file.

- Created #482 

Closes #478

### CODE REVIEW

File: src/helpers/showFilterUtils.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ const filterShowsByGenre = (showData: ShowData[], genreId: number): ShowData[] =
 const filterShowsByType = (showData: ShowData[], showType: 'movie' | 'tv'): ShowData[] => {
```
Comment (maxh1231) [inline src/helpers/showFilterUtils.ts:25]: @Thenlie Should we change this parameter to `media_type` to match the new naming convention?

**Review by maxh1231 — APPROVED** [review]
Nice work!

File: src/helpers/showFilterUtils.ts (hunk 1 above)
Comment (Thenlie) [reply src/helpers/showFilterUtils.ts:25]: I think using camelCase for function params is still best. Just using snake case for json responses from TMDB.

Comment (maxh1231) [reply src/helpers/showFilterUtils.ts:25]: That's what I meant, `mediaType` or something of that nature. Just curious

Comment (Thenlie) [reply src/helpers/showFilterUtils.ts:25]: Yeah that is definitely a reasonable idea to keep the naming more consistent.

**Pull request merged**